docker-hub/download-rate-limit.md

```diff
@@ -61,17 +61,17 @@ To get a token with a user account (if you are authenticating your pulls) - don'
 $ TOKEN=$(curl --user 'username:password' "https://auth.docker.io/token?service=registry.docker.io&scope=repository:ratelimitpreview/test:pull" | jq -r .token)
 ```
 
-Then to get the headers showing your limits, run the following (keep in mind that requesting a manifest emulates a pull and will count against the limits):
+Then to get the headers showing your limits, run the following:
 
 ```
-$ curl -v -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest 2>&1 | grep RateLimit
+$ curl -I -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest 2>&1 | grep RateLimit
```

Comment:
Consider using `--head` rather than `-I`, it's more explicit to the readers that it is a head request

Reply:
What if we print out something informational if the RateLimit info isn't found?

This seems to work:

```
curl --head -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest | grep RateLimit || echo "No headers found (unlimited pulls)"

```